programs/drift/src/state/perp_market.rs

```diff
@@ -307,6 +316,12 @@ impl PerpMarket {
         PerpOperation::is_operation_paused(self.paused_operations, operation)
     }
 
+    pub fn can_skip_auction_duration(&self) -> DriftResult<bool> {
+        Ok((self.amm.net_revenue_since_last_funding > 0
+            && self.amm.amm_lp_allowed_to_jit_make(true)?)
```

Comment:
is this call expensive for cus? should we try to cache it?

Reply:
yeah i will leave that as a todo